README.md

```diff
@@ -247,3 +247,9 @@ You can log into your instance with you login.gov user.
 Continuous Integration via [GitHub Actions](https://github.com/GSA/catalog.data.gov/actions/workflows/commit.yml).
 
 Continuous Deployment via [GitHub Actions](https://github.com/GSA/catalog.data.gov/actions/workflows/publish.yml).
+
+## Note on DB migration
+In the CKAN database we have added an additional index on column `resource_id` in table `resource_view`. So on next major release before we upgrade DB by running command `ckan db upgrade`, we need to manually remove the index first to avoid any SQL error.
+```
+DROP INDEX idx_view_resource_id;
```

Comment:
Is this the only index we made?

Reply:
This is the only index added into the CKAN core. We also added an index in harvest extension but that is not the part of core db migration process.